marathon_acme/tests/test_clients.py

```diff
@@ -665,6 +666,9 @@ def test_get_events(self):
         request.finish()
         yield d
 
+        # Expect request.finish() to result in a logged failure
+        flush_logged_errors(ResponseDone)
```

Comment:
Does this assert that there was a logged error to flush (as the comment suggests)? If not, do we want to make that assertion?